examples/java/hello_world/MainActivity.java

```diff
@@ -104,8 +104,10 @@ private void makeRequest() {
               return Unit.INSTANCE;
             })
             .onError((error) -> {
-              recyclerView.post(
-                  () -> viewAdapter.add(new Failure("failed with error " + error.getMessage())));
+              String msg = new String("failed with error after " + error.getAttemptCount() +
+                                      " attempts: " + error.getMessage());
+              Log.d("MainActivity", msg);
```

Comment:
```suggestion
              Log.d("MainActivity", "failed with error after " + error.getAttemptCount() +
                                      " attempts: " + error.getMessage());
```

Reply:
I want to give the Failure object the same string, though

Reply:
uhh how about
```suggestion
              String msg = "failed with error after " + error.getAttemptCount() +
                                      " attempts: " + error.getMessage();
              Log.d("MainActivity", msg);
```
Since the `new String()` is redundant here